pkg/plugins/validation.go

```diff
@@ -0,0 +1,122 @@
+package plugins
+
+import (
+	"embed"
+	"path"
+
+	"github.com/ghodss/yaml"
+	"github.com/xeipuuv/gojsonschema"
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/proto"
+
+	"github.com/kumahq/kuma/pkg/core/validators"
+)
+
+// we only need schema.yaml but during make check we delete all of them and that makes compilation fail
```

Comment:
I don't like this, why can't I just simply have a match that doesn't add any files? :/